cmd/command/sp_exit.go

```diff
@@ -0,0 +1,63 @@
+package command
+
+import (
+	"github.com/bnb-chain/greenfield-storage-provider/cmd/utils"
+	virtualgrouptypes "github.com/bnb-chain/greenfield/x/virtualgroup/types"
+
+	"github.com/urfave/cli/v2"
+)
+
+const spExitCommands = "Sp Exit Commands"
+
+var SpExitCmd = &cli.Command{
+	Action: SpExitAction,
+	Name:   "spExit",
+	Usage:  "sp exit",
+	Flags: []cli.Flag{
+		utils.ConfigFileFlag,
+	},
+	Category:    spExitCommands,
+	Description: ``,
+}
+
+func SpExitAction(ctx *cli.Context) error {
+	cfg, err := utils.MakeConfig(ctx)
+	if err != nil {
+		return err
+	}
+
+	msg := &virtualgrouptypes.MsgStorageProviderExit{
+		StorageProvider: cfg.SpAccount.SpOperatorAddress,
+	}
+
+	spClient := utils.MakeGfSpClient(cfg)
+	_, err = spClient.SpExit(ctx.Context, msg)
+	return err
+}
+
+var CompleteSpExitCmd = &cli.Command{
+	Action: CompleteSpExitAction,
+	Name:   "completeSpExit",
+	Usage:  "complete sp exit",
+	Flags: []cli.Flag{
+		utils.ConfigFileFlag,
+	},
+	Category:    spExitCommands,
+	Description: ``,
```

Comment:
ditto

These descriptions looks more significant as they can help user understand when and why they need to use these cmd. 

E.g.  we said "If the SP has no more GVGF/GVG associated, anyone can send a tx with MsgCompleteStorageProviderExit to complete such SP's exit. "  
So, who actually need to invoke CompleteSpExitCmd, and how to construct a "config file"?  do we have some example for using these cmd?
By considering 2 cases:
1. The successor SP  sends a tx with MsgCompleteStorageProviderExit
2. The exiting SP  sends a tx with MsgCompleteStorageProviderExit
These 2 cases will use different config file to achieve the same result right ?